components/button/button.css

```diff
@@ -0,0 +1,130 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+:root{
+    --color-primary : #FF3B30;
+    --color-secondary: grey;
+    --color-success: green;
+    --color-danger: red;
+    --color-warning: rgb(218, 218, 54);
+
+    --color-white: #Fff6ea;
+    --color-black: #282828;
+}
+
+button {
+    background-color: transparent;
+    border: none;
+    cursor: pointer;
+    outline: none;
+}
+
+.button-container{
+    margin: 1rem;
+}
+
+.btn {
+    background-color: none;
+    color: var(--color-white);
+    padding: 0.5rem 1rem;
+
+}
+.btn-large{
+    padding: 1rem 4rem;
+}
+.btn:hover{
+    background-color: var(--color-black);
+    /* opacity: 0.7; */
+}
+
+.btn-primary{
+    background-color: var(--color-primary);
+}
+.btn-secondary{
+    background-color: var(--color-secondary);
```

Comment:
secondary button generally doesn't have background color, they have border color

Reply:
I took inspiration from bootstrap [https://getbootstrap.com/docs/4.0/components/buttons/]() while designing buttons, and I saw the secondary button there having background color as grey. Do I need to change this?